core/src/streams.rs

```diff
@@ -1189,6 +1224,168 @@ impl<'gc> NetStream<'gc> {
             }
         }
 
+        if let Some(NetStreamType::F4v {
+            context: media_context,
+            frame_id,
+            video_stream,
+        }) = &mut write.stream_type
+        {
+            println!("frame id: {:?}, end_time: {}", frame_id, end_time);
```

Comment:
Seems to now be called `max_time`: https://github.com/ruffle-rs/ruffle/blob/df0ac4395b84be317e39271a8f5a1c7d6227ada4/core/src/streams.rs#L1110